tests/antd.test.tsx

```
@@ -28,6 +32,28 @@ Object.defineProperty(window, "ResizeObserver", {
     })),
 })
 
+Object.defineProperty(global, "MessageChannel", {
+    writable: true,
+    value: class MessageChannel {
+        port1 = {
+            onmessage: null as any,
+            postMessage: (data: any) => {
+                if (this.port2.onmessage) {
+                    setTimeout(() => this.port2.onmessage({data}), 0);
+                }
+            },
+        };
+        port2 = {
+            onmessage: null as any,
+            postMessage: (data: any) => {
+                if (this.port1.onmessage) {
+                    setTimeout(() => this.port1.onmessage({data}), 0);
+                }
+            },
+        };
+    },
+})
+
```

The goal wasn't to hide the errors and warnings but to fix the usage of `act` or other Jest component usages (assuming they aren't used properly).